### PR TITLE
New package: pev-0.70

### PR DESCRIPTION
**Pull request by diogoleal**

### CODE REVIEW

**Comment by beefcurtains** [comment]
:+1: looks good

File: srcpkgs/pev/template [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Template file for 'pev'
+pkgname=pev
+version=0.70
+revision=1
+wrksrc=${pkgname}
+build_style=gnu-makefile
+makedepends="libressl-devel pcre-devel glibc-devel"
```
Comment (beefcurtains) [inline srcpkgs/pev/template]: ``glibc-devel`` is redundant, I think

Comment (diogoleal) [inline srcpkgs/pev/template]: ```glibc-devel``` removed

**Comment by beefcurtains** [comment]
Not sure if ``/usr/lib/libpe.so`` should be in there, if it's possible to create a ``-devel`` with header files it should go in there. Other than that it's good to go.

**Comment by diogoleal** [comment]
this is not necessary. the ```libpe``` is a library part. I'll create the package soon.
https://github.com/merces/libpe

**Comment by beefcurtains** [comment]
Feel free to pull that whenever.

**Pull request merged**